seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/java/org/apache/seatunnel/engine/e2e/joblog/JobLogIT.java

```diff
@@ -103,6 +104,9 @@ public void testJobLogFile() throws Exception {
 
         assertConsoleLog();
         assertFileLog();
+        assertFileLogClean(false);
```

Comment:
I think we need to add examples of multi-parallelism jobs and have this job run on a multi-node cluster.